scss/tech.scss

```
@@ -32,6 +32,11 @@
     width: 100%;
     z-index: -1; 
   }
+
+  &:hover::before {
+    transition-property: all;
```

This should go in `&::before` and replace by
```suggestion
    transition: transform 330ms;
```